techniques/systemSettings/userManagement/userManagement/7.0/metadata.xml

```diff
@@ -146,8 +147,25 @@ It is intended to check the user parameters on the target host.
           <DESCRIPTION>Password for this account</DESCRIPTION>
           <CONSTRAINT>
             <MAYBEEMPTY>true</MAYBEEMPTY>
-            <TYPE>password</TYPE>
+            <TYPE>masterPassword</TYPE>
             <PASSWORDHASH>linux-shadow-md5,linux-shadow-sha256,linux-shadow-sha512,plain</PASSWORDHASH> 
+            <!-- 
+             Tell that master password must create other variables derived from the user input from
+             that one. The accepted values for now are "aix" and "linux" (or both, comma separated). 
+             The derived variable name will the current name postfixed with _AIX (or _LINUX)
+
+             A correspondance is made between hash algo listed above and the matching one on target OS:
+             Linux md5 crypt is mapped to AIX "smd5" version, Linux Sha-Crypt-256 is
+             mapped to AIX ssha256, and Linux Sha-Crypt-512 to AIX ssha512.
+             AIX ssha256 and ssha512 need the JCE PBKDF2WithHmacSHA256 / PBKDF2WithHmacSHA512.
+             They are provided on Oracle Java 8 JVM standard installation, but NOT in
+             Java 7 and some other vendor versions.
+             In case these algo are not available, a fallback to AIX ssha1 (which uses
+             PBKDF2WithHmacSHA1) will be done. This hash scheme is also quite robust, but
+             if you want maximum security, you must use for Rudder a JVM which provides the higher
+             level algo, like Open JDK 8
+             -->
+            <DERIVE>aix</DERIVE>
```

Comment:
Why don't we have `linux` in `<DERIVE>` here? The comment says this is needed to create the password for linux (postfixed with LINUX).
What I understand for the current code is that it will only generate the password for AIX.